README.md

```diff
@@ -1,137 +1,75 @@
-<div align="center">
-<h1 align="center">Unreal Style Sheet</h1>
-
-  <p align="center">
-    A Qt style sheet designed to make your tools look native in Unreal Engine 5
-  </p>
-</div>
-
-## About The Project
-
-<br>
-
-<div align="center">
-<img src="https://i.imgur.com/y7UP7k3.jpg" alt="main" width="100%"/>
-</div>
-
-<br>
-
-This side project is my personal attempt to make Qt application feels 
-more natural in Unreal Engine 5.
-The style sheet covers some of the most common used widgets, and tries to
-match the Unreal native UI as close as possible.
-
-If you feel there's anything missing, feel free to submit an issue or a PR.
-
-> Unreal has a different UI framework (Slate), future [read](https://minimaleffort.tech/qt-to-slate-transition-guide/) on comparing Qt to Slate 
-
-## Getting Started
-
-### Using the style sheet
-
-1. Simply grab the style sheet file `ue.qss` and resource file `icons.rcc`
-
-
-2. Launch your Qt application and link the files
-
-   ```python
-   if __name__ == '__main__':
-       app = QtWidgets.QApplication(sys.argv)
-   
-       QtCore.QResource.registerResource("PATH_TO_ICONS.RCC")
-   
-       with open("PATH_TO_UE.QSS", 'r') as f:
-           qss = f.read()
-           app.setStyleSheet(qss)
-   
-       window = YourTool()
-       window.show()
-   
-       sys.exit(app.exec_())
-
-   ```
-
-### Comparison
-
-The following `.ui` files for testing can be found inside the `/ui` folder.
-
-<table>
-   <tr>
-      <td>
-         <p>default (editor.ui)</p>
-         <img src="https://i.imgur.com/EX5naMy.png" alt="editor-qt"/>
-      </td>
-   </tr>
-   <tr>
-      <td>
-         <p>ue.qss</p>
-         <img src="https://i.imgur.com/M8CU4cN.jpg" alt="editor-ue"/>
-      </td>
-   </tr>
-</table>
-
-<table>
-   <tr>
-      <td>
-         <p>default (tree.ui)</p>
-         <img src="https://i.imgur.com/Xk8XYlQ.png" alt="tree-qt"/>
-      </td>
-   </tr>
-   <tr>
-      <td>
-         <p>ue.qss</p>
-         <img src="https://i.imgur.com/DcQQwak.jpg" alt="tree-ue"/>
-      </td>
-   </tr>
-</table>
-
-<table>
-   <tr>
-      <td>
-         <p>default (progress.ui)</p>
-         <img src="https://i.imgur.com/6yMuCKD.png" alt="progress-qt"/>
-      </td>
-   </tr>
-   <tr>
-      <td>
-         <p>ue.qss</p>
-         <img src="https://i.imgur.com/NyskX8m.jpg" alt="progress-ue"/>
-      </td>
-   </tr>
-</table>
+# Unreal Stylesheet [![PyPI](https://img.shields.io/pypi/v/unreal_stylesheet?color=blue)](https://pypi.org/project/unreal-stylesheet/)
 
+ A Qt stylesheet designed to make your tools look native in Unreal Engine 5  
 
-### Modifying the style sheet
+```python
```

Comment:
one last comment, let's move this code block the installation section as "example", also I expanded it so it's more clear:
```
import unreal_stylesheet


if __name__ == '__main__':
    app = QtWidgets.QApplication(sys.argv)

    unreal_stylesheet.setup()

    window = MainWindow()
    window.show()
    sys.exit(app.exec_())
```

Reply:
updated the README, have a look [here](https://github.com/hannesdelbeke/unrealStylesheet)